ra/ra.go

```diff
@@ -2210,6 +2236,17 @@ func (ra *RegistrationAuthorityImpl) AdministrativelyRevokeCertificate(ctx conte
 	}
 
 	err = ra.revokeCertificate(ctx, serialInt, issuerID, revocation.Reason(req.Code))
+	// Perform an Akamai cache purge to handle occurrences of a client
+	// successfully revoking a certificate, but the initial cache purge failing.
+	if errors.Is(err, berrors.AlreadyRevoked) {
+		if cert != nil {
+			// An administrator will most likely want to know why a purge failed.
+			err = ra.purgeOCSPCache(ctx, cert, issuerID)
+			if err != nil {
+				return nil, err
```

Comment:
I'd return a more descriptive error, so that the admin-revoker operator is explicitly aware that both the revocation itself _and_ the cache-purge failed, and they should probably try again.